test/wasm/rot.js

```diff
@@ -0,0 +1,14 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft Corporation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+const blob = WScript.LoadBinaryFile('rot.wasm');
+const moduleBytesView = new Uint8Array(blob);
+var a = Wasm.instantiateModule(moduleBytesView, {}).exports;
+print(a.rotl(11,2)); // == 44
```

Comment:
Actually, I do have a baseline file and an rlexe.xml, but I haven't included those because I still haven't gotten around to scripting the encoding of the wast into wasm.